develop/tutorials/articles/02-from-liferay-6-to-liferay-7/07-upgrading-plugins-to-liferay-7/07-upgrading-application-upgrade-and-verifier-processes.markdown

```diff
@@ -0,0 +1,384 @@
+# Upgrading Application Upgrade and Verifier Processes
+
+Since Liferay 7.0 and Lifery DXP, a new upgrade framework and verifier framework 
+is available for you to use for your application. You can upgrade your existing 
+upgrade and verify processes to use the new framework in just a few steps.
+
+This tutorial demonstrates how to:
+
+- Upgrade an existing upgrade process to the new upgrade framework
+- Upgrade an existing verify process to the new verify framework
+
+Go ahead and get started with the upgrade process next.
+
+## Upgrading Your Upgrade Process to the New Framework
+
+Follow the steps below to migrate your code to the new framework.
+
+1.  If your application has any dependencies, add a dependency on the 
+    `portal-upgrade` module to your `build.gradle` file:
+    
+        provided group: "com.liferay", name: "com.liferay.portal.upgrade", 
+        version: "2.0.0"
+
+2.  Check your database schema version against your bundle version.
+
+    If the database schema is in a different version than the bundle version, 
+    specify the different versions for the bundle and schema in your `bnd.bnd` 
+    file, using the `require-SchemaVersion` property:
+
+        require-SchemaVersion: 1.0.2
+
+    If no `Required-SchemaVersion` is found, the `Bundle-Version` header will be 
+    used. Now that the build files are configured, you can move on to the 
+    upgrade class next.
```

Comment:
I'm confused by what these different versions refer to.

It would be helpful to demonstrate how to check these version number and give an example (perhaps one in existing Liferay code) in which the database schema version differs from the bundle version (module version?).

Thanks!